R/get_message_data.R

```diff
@@ -1,19 +1,25 @@
 get_message_data = function(
   dir = ".",
   custom_translation_functions = list(R = NULL, src = NULL),
+  use_conditions = TRUE,
+  use_conditions_f = TRUE,
```

Comment:
I'm not liking the `_f` here... do we think keeping the toggle separate has a lot of value? e.g. a use case for keeping formatted conditions but not "regular" ones, or vice versa?

If so, what about `use_conditions = c(TRUE, FALSE)` instead? It's more obscure/less discoverable but keeps the more common default easier to understand and doesn't introduce this potentially confusing `_f` shorthand.

WDYT

Reply:
If you like the `style` idea, I could just push it down to here. I don't think it's really necessary to break up into 3 separate binary variables.

Reply:
yes, let's go with that for now, i think it's easiest to give the user only one lever to push here